src/operator/contrib/bilinear_resize-inl.cuh

```diff
@@ -60,19 +60,63 @@ static unsigned getNumThreads(int nElem, const bool smaller) {
   return smaller ? (MAX_BLOCK_SIZE >> 1) : MAX_BLOCK_SIZE;
 }
 
-// caffe_gpu_interp2_kernel overloading with Tensor<xpu, 3, DType>
+__device__ __forceinline__ size_t
+idx(const size_t nc,
+  const size_t height,
+  const size_t width,
+  const size_t y,
+  const size_t x) {
+  return (nc * height + y) * width + x;
+}
+
+template <typename Acctype>
+__host__ MSHADOW_XINLINE static Acctype cu_area_pixel_compute_scale(
```

Comment:
We can remove `__host__`, since `#define MSHADOW_XINLINE MSHADOW_FORCE_INLINE __device__ __host__
`